templates/settings.py.erb

```diff
@@ -38,8 +38,11 @@ REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES = (
     'pulpcore.app.authentication.PulpRemoteUserAuthentication'
 )
 
-<%# This setting whitelists paths that can be used for repository sync with file protocol. Katello uses the path /var/lib/pulp/sync_imports/ to run tests -%>
+<%# These settings specify paths that can be used for repository sync with file protocol, -%>
+<%# and for content view version import/export. -%>
+<%# Katello uses the path /var/lib/pulp/sync_imports/ to run tests -%>
```

Comment:
I think these comments are not that good. The settings are documented [here](https://docs.pulpproject.org/pulpcore/settings.html#allowed-import-paths) and [here](https://docs.pulpproject.org/pulpcore/settings.html#allowed-export-paths) which is much more important than what Katello does with it.

Reply:
@ekohl I was just updating the comment so it would remain accurate.  But I don't have a strong opinion about the proper place for Katello-specific comments, so I'll go ahead and just remove the comment entirely.  would you be satisfied with that?

Reply:
Yes, that's fine by me.